terraform/shared/video-player/js/common.js

```diff
@@ -21,6 +21,42 @@ var live_ipfs = getURLParam('live') // Set m3u8 file URL to override IPFS live s
 var vod_ipfs = getURLParam('vod')   // Set IPFS content hash of mp4 file to play IPFS on-demand video stream
 var startFrom = getURLParam("startFrom"); // Timecode to start video playing from

```

Comment:
- Can we simplify the URL param as `start` or `from` or `bookmark`?
- The js var should be `start_from` casing, see line above